packages/flex-plugin-scripts/src/scripts/deploy.ts

```diff
@@ -232,6 +242,20 @@ export const _doDeploy = async (nextVersion: string, options: Options): Promise<
     return data;
   });
 
+  const deployResult = {
+    serviceSid: runtime.service.sid,
+    accountSid: runtime.service.account_sid,
+    environmentSid: runtime.environment.sid,
+    domainName: runtime.environment.domain_name,
+    isPublic: options.isPublic,
+    nextVersion,
+    pluginUrl,
+  };
+
+  if (routeCollision && !options.overwrite) {
+    return deployResult;
+  }
+
   // Register service sid with Config service
   await progress('Registering plugin with Flex', async () => {
```

Comment:
Mm, we probably still want to do this step anyway - this is backfill script. Move this before the return on the previous line.